blog/authors.yml

```diff
@@ -17,3 +17,14 @@ eddyb:
     /img/authors/eddyb.png
   socials:
     github: eddyb
+
+schell:
+  name: Schell Carl Scivally
+  title: Rust GPU and Renderling maintainer
+  url: https://github.com/schell
```

Comment:
```suggestion
```

Reply:
This is jank, but docusaurus doesn't need this as it will use github if the account is set by default.